spec/requests/controllers/jams_controller_spec.rb

```diff
@@ -56,6 +56,31 @@
       end.to_not change{ user.notifications.count }
     end
 
+    context 'tagging' do
+      let(:jam_params) do
+        {
+          bpm: '100',
+          song_key: 'A Major',
+          jam_type: 'MIX',
+          style_list: 'Electronic, Lofi',
+          could_use_list: 'Bass, Guitar, Vocals',
+          file: file }
```

Comment:
```
   file: file,
}
```

Reply:
BT loves the trailing comma in the list, rubocop yells at you though